test-validator/src/lib.rs

```diff
@@ -107,9 +107,14 @@ pub struct TestValidatorGenesis {
     pub max_genesis_archive_unpacked_size: Option<u64>,
     pub accountsdb_plugin_config_files: Option<Vec<PathBuf>>,
     pub accounts_db_caching_enabled: bool,
+    inactivate_feature_list: Option<Vec<Pubkey>>,
 }
 
 impl TestValidatorGenesis {
+    pub fn inactivate_features(&mut self, inactivate_list: Vec<Pubkey>) -> &mut Self {
+        self.inactivate_feature_list = Some(inactivate_list);
```

Comment:
To save a future the caller some grief, wdyt about adding check to ensure the provided features are known, by checking the `FEATURE_NAMES` HashMap here?  

Otherwise if caller provides a non-feature account, `genesis_config.accounts.remove(invalidate_feature);` is going to just delete any account.

Reply:
Can do, familiar with the FEATURE_NAMES.

Reply:
Changed in fb8b5b5. However; checking in `initialize_ledger` and ignoring non-Feature PKs